tools/releaseBuild/azureDevOps/templates/release-PublishPackageMsftCom.yml

```
@@ -9,19 +14,40 @@ steps:
   displayName: Set Package version
 
 - pwsh: |
-   git clone https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'
+    Write-Verbose -verbose "git clone --verbose https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'"
```

this might be better as:
```suggestion
$gitArgs = "clone",
 "--verbose", 
 "https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools", 
 '$(Pipeline.Workspace)/tools'
$gitArgs | Write-Verbose -Verbose
git $gitArgs
```